jbpm-services/jbpm-services-api/src/main/java/org/jbpm/services/api/RuntimeDataService.java

```diff
@@ -206,8 +206,14 @@ public int getValue() {
      * @return A list of {@link ProcessInstanceDesc} instances representing the available process instances.
      */
     Collection<ProcessInstanceDesc> getProcessInstancesByParent(Long parentProcessInstanceId, List<Integer> states, QueryContext queryContext);
-    
-    
+
+    /**
+     * Returns list of process instance descriptions
+     * @param processInstanceId id of the parent process instance
```

Comment:
still missing params states and queryContext in the java doc. Would you please add them?